docusaurus/docs/configuration/ort-yml.md

```diff
@@ -75,8 +75,8 @@ excludes:
 ```
 
 Where the list of available options for `reason` is defined in
-[PathExcludeReason.kt](https://github.com/oss-review-toolkit/ort/blob/main/model/src/main/kotlin/config/PathExcludeReason.kt). For how to write a glob pattern, please
-see the
+[PathExcludeReason.kt](https://github.com/oss-review-toolkit/ort/blob/main/model/src/main/kotlin/config/PathExcludeReason.kt).
```

Comment:
Can you briefly explain in the commit message how lint issues could have been added if the files were mostly just copied?